stdlib/3/pathlib.pyi

```diff
@@ -31,6 +31,8 @@ class PurePath(_PurePathBase):
     def __gt__(self, other: PurePath) -> bool: ...
     def __ge__(self, other: PurePath) -> bool: ...
     def __truediv__(self: _P, key: Union[str, PurePath]) -> _P: ...
+    if sys.version_info[0] == 2:
```

Comment:
Minor nit: In typeshed we use the form `sys.version_info < (3,)` (or similar) consistently. I would like to keep it this way.